sdk/cognitiveservices/azopenai/README.md

```diff
@@ -19,7 +19,7 @@ Azure OpenAI Service provides access to OpenAI's powerful language models includ
 Install the `azopenai` and `azidentity` modules with `go get`:
 
 ```bash
-go get github.com/Azure/azure-sdk-for-go/sdk/cognitiveservices/azopenai
+go get github.com/Azure/azure-sdk-for-go/sdk/ai/azopenai
```

Comment:
I don't think we need to make any of these changes in the old module.

Reply:
I'm actually tempted to leave this one. Just in case someone cuts and pastes from some randomly found readme.

Reply:
It just seems weird that the readme for module A says "go get module B".

Reply:
I ended up just clearing the readme out to make it obvious this is the old module (ie, we're really only publishing this so users get a pointer to the new module anyways).